daprdocs/content/en/developing-applications/building-blocks/actors/namespaced-actors.md

```diff
@@ -0,0 +1,41 @@
+---
+type: docs
+title: "Namespaced actors"
+linkTitle: "Namespaced actors"
+weight: 40
+description: "Learn about the multi-tenant placement service in actors"
+---
+
+Namespaced actors use the multi-tenant placement service. With this service, in a scenario where each tenant has its own namespace, sidecars belonging to a tenant named "Tenant A" won't receive placement information for "Tenant B". 
```

Comment:
Namespacing in Dapr provides isolation and hence multi-tenancy. Dapr supports namespaced actors. This means the same actor type can be deployed into different namespaces and instances of actors can call across the namespaces. Note that the namespace is not part of the actor name (that is an actor id), namespace is used for the isolation of the deployed actor services. 

On Kubernetes you use namespaces when deploying the actor applications. For example

kubectl create namespace namespace-actorA
kubectl config set-context --current --namespace=namespace-actorA

Then deploy your actor applications into this namespace.

In self hosted mode you can specify the namespace for a Dapr instance by setting the NAMESPACE environment variable.